### ISSUE TITLE
Document on rejectUnsupportedCards

### ISSUE THREAD

**Issue by maxisam** [braintree/braintree-web#433]
in document 
```javascript
braintree.hostedFields.create({
  client: clientInstance,
  fields: {
    number: {
      selector: '#card-number'
    },
    cvv: {
      selector: '#cvv',
      placeholder: '•••'
    },
    expirationDate: {
      selector: '#expiration-date',
      type: 'month'
    }
  },
  rejectUnsupportedCards: true
}, callback);
```

But there is no rejectUnsupportedCards for options. It is actually in HostedFieldsField options

**Comment by crookedneighbor** [comment]
Thanks for noticing this. Want to open up a PR? Otherwise, we'll get this fixed up soon.

**Comment by maxisam** [comment]
I will let you guys handle it. thx

**Comment by crookedneighbor** [comment]
:+1:

**Comment by crookedneighbor** [comment]
The fix is out in 3.46.0's documentation.

In addition, we've introduced a new `supportedCardBrands` object where you explicitly turn on or turn off cards from your merchant configuration.


```js
braintree.hostedFields.create({
  client: clientInstance,
  fields: {
    number: {
      selector: '#card-number',
      supportedCardBrands: {
        visa: false, // prevents Visas from showing up as valid even when the Braintree control panel is configured to allow them
        'diners-club': true // allow Diners Club cards to be valid (processed as Discover cards on the Braintree backend)
      }
    },
```
